### PR TITLE
Fix multi-currency issue with Pay by link payment form

### PR DESCRIPTION
**Pull request by shahbaz-mehar**

### CODE REVIEW

**Review by BushraAsif — COMMENTED** [review]
Changes reviewed

File: Test/integration-tests/cypress/e2e/examples/payment.cy.js [hunk 1]
```diff
@@ -376,8 +376,8 @@ describe('Payments', function () {
 
         })
     })
-
-    it('CC Pay by link', function () {
+    //skipping this test
+    it.skip('CC Pay by link', function () {
```
Comment (emicha) [inline Test/integration-tests/cypress/e2e/examples/payment.cy.js:380]: Why are you skipping the tests?

File: Test/integration-tests/cypress/e2e/examples/payment.cy.js [hunk 2]
```diff
@@ -414,8 +414,8 @@ describe('Payments', function () {
             })
         })
     })
-
-    it('Klarna Pay by link', function () {
+    //skipping this test
+    it.skip('Klarna Pay by link', function () {
```
Comment (emicha) [inline Test/integration-tests/cypress/e2e/examples/payment.cy.js:418]: Why are you skipping the tests?

File: Test/integration-tests/cypress/e2e/examples/payment.cy.js (hunk 1 above)
Comment (shahbaz-mehar) [reply Test/integration-tests/cypress/e2e/examples/payment.cy.js:380]: @emicha Due to a version change in Magento, these tests are not functioning as expected. Taimur is currently debugging the issue. Once resolved, we will incorporate these tests as well. But overall functionality is working fine.

Comment (shahbaz-mehar) [reply Test/integration-tests/cypress/e2e/examples/payment.cy.js]: @emicha The tests have been fixed and reverted.

File: Test/integration-tests/cypress/e2e/examples/payment.cy.js (hunk 2 above)
Comment (shahbaz-mehar) [reply Test/integration-tests/cypress/e2e/examples/payment.cy.js]: @emicha The tests have been fixed and reverted.

**Review by emicha — APPROVED** [review]

**Pull request merged**